### PR TITLE
Imported decks should have cleaner names now.

### PR DESCRIPTION
**Pull request by aldob02**
see title

### CODE REVIEW

File: src/main/java/data/DeckPathCleaner.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package data;
+
+public class DeckPathCleaner {
+    /**
+     * Removes path and extension from a file
+     * @param filePath where the file is located
+     * @return clean name of the file
+     */
+    public static String cleanName(String filePath){
```
Comment (AndrewNQN) [inline src/main/java/data/DeckPathCleaner.java:9]: Much needed function. Only thing I could see going wrong is if people put '\' into their deck name. However, I think that we can implement these changes. Please include in the ReadMe the limitations on deckNames

**Review by AndrewNQN — APPROVED** [review]
I left a comment outlining my concern, but otherwise, everything looks good. Thank you for these swift changes.

File: src/main/java/data/DeckPathCleaner.java (hunk 1 above)
Comment (aldob02) [reply src/main/java/data/DeckPathCleaner.java:9]: I'll can make it so it removes any quotation marks in the name while importing

Comment (aldob02) [reply src/main/java/data/DeckPathCleaner.java:9]: Actually nevermind I think I know how to include quotations without messing anything

Comment (aldob02) [reply src/main/java/data/DeckPathCleaner.java:9]: image
actually no change needed xD

Comment (AndrewNQN) [reply src/main/java/data/DeckPathCleaner.java:9]: Perfect

**Review by tsoisze2 — APPROVED** [review]
Everything looks good. Every class is short and succinct. Maybe need to add some documentation headers.

**Review by jakan9 — APPROVED** [review]
Needs documentation for easier understanding, else looks great!

**Pull request merged**